db/schema.rb

```diff
@@ -56,7 +71,9 @@
     t.datetime "updated_at", precision: 6, null: false
     t.bigint "payment_intent_id", null: false
     t.boolean "refunded", default: false
+    t.bigint "purchaser_id"
```

Comment:
This doesn't have to be nonnullable bc maybe we'll one day combine purchased gift cards into one for organizations to bulk order